verilog/formatting/tree_unwrapper.cc

```diff
@@ -972,6 +972,7 @@ void TreeUnwrapper::SetIndentationsAndCreatePartitions(
     }
 
       // Add a level of grouping that is treated as wrapping.
+    case NodeEnum::kPackageImportList:
```

Comment:
Placed here, this "wraps" which means indenting 4 spaces `style_.wrap_spaces`.
Let's indent this only 2 spaces, look for `style_.indent_spaces`.

(Test case will need a minor adjustment.)

Reply:
Thanks for the clarification. 
I thought it needed to match the indentation level of the module i/o ports. That's why I used an indentation of 4 spaces.

Reply:
To be fair, there is disagreement over 2 vs. 4 spaces in some contexts.
In such cases, we've surfaced a few knobs to the users who need them.

See uses of `IndentationStyle` in here:
https://cs.opensource.google/verible/verible/+/master:verilog/formatting/format_style.h

and some of their corresponding uses here:
https://cs.opensource.google/verible/verible/+/master:verilog/tools/formatter/verilog_format.cc

No need to address this now, we can add such a style option and flag as a follow-up later.